tests/acceptance/features/apiSpaces/listSpaces.feature

```diff
@@ -55,7 +55,7 @@ Feature: List and create spaces
       | quota@@@total    | 2000          |
       | root@@@webDavUrl | %base_url%/dav/spaces/%space_id% |
 
-  Scenario: Alice creates folder via Graph api in space, she expects a 201 code and she checks that folder exists
+  Scenario: Alice creates a folder via the Graph api in space, she expects a 201 code and she checks that folder exists
```

Comment:
Maybe we should move it to `uploadSpaces`?

Reply:
good idea